server/i18n/ja.yml

```diff
@@ -0,0 +1,37 @@
+archived: ""
+comment already exist in this thread: ""
+comment does not exist in this thread: ""
+createdBy is required: ""
+duplicated value: ""
+encoding password: ""
+file is required: ""
+file not found: ""
+file size cannot be zero: ""
+invalid alias: ""
+invalid default values: ""
+invalid email: ""
+invalid field: ""
+invalid key: ""
+invalid password: ""
+invalid role: ""
+invalid type: ""
+invalid user name: ""
+invalid value: ""
+invalid values: ""
+invalid workspace name: ""
+items cannot be empty: ""
+max must be larger then min: ""
+password at least 8 characters: ""
+password should have lower case letters: ""
+password should have numbers: ""
+password should have upper case letters: ""
+personal workspace cannot be modified: ""
+projectID is required: ""
+target user does not exist in the workspace: ""
+thread is required: ""
+title cannot be empty: ""
+unsupported entity: ""
+user already joined: ""
+uuid is required: ""
+value is required: ""
+workspace id is required: ""
```

Comment:
Filling them will be done at another PR, right?

Reply:
I guess Baba san will work on that